oauth-tunnel-client.rb

```diff
@@ -7,8 +7,8 @@ def ext
 
   desc 'Create a secure local proxy with Shopify GCP services'
   homepage 'https://github.com/Shopify/oauth-tunnel-client'
-  url 'https://storage.googleapis.com/oauth-tunnel-binaries/oauth-tunnel-client-binaries-99d32760b2433ca850d0b6f927ca77b1b1ecb06f.tar.gz?GoogleAccessId=pipa-production@shopify-docker-images.iam.gserviceaccount.com&Expires=1542043420&Signature=nizLTr6sCeVp5%2F92ftVYdRDphme%2BpnSfjzi1mcNRZaimQv%2FyZH%2Fp%2BvEe4cVOEeYspz%2Fko%2FxKqgjyp%2FDNCZY98BDGZMWdJPOBv8d1cbgufZSgzHf3e%2FmBDu3zkMewGPWgtX1cx9VNQ5vE6R7qcR%2BdVRDYQGFnEiS0HldY95bYPclThuGFMxitHfzm9oqLabOXJWN5k4xjMnVaImRuR2ROIxjDwSIOIE3TkwLoN2k2rxbAoc7nPcIZR6GO4ccggVmXk6EEMmxN4NG7xZP3w04XID%2F12IwJdO1SDfAgwXJZ8JVmiQWG2RaKggFVEvmOvKaI5aJDnIshxDShmv%2BPY8vkKQ%3D%3D', using: GoogleStorageDownloadStrategy
```

Comment:
can probably delete unused `GoogleStorageDownloadStrategy` too while we're at this?